### DISCUSSION TITLE
Change row color?

### DISCUSSION THREAD

**Discussion by jmarshall9120** [HC200ok/vue3-easy-data-table#296] [Q&A]
Is there a way to change row color by content?  I would assume this is easiest done with some sort of item-slot that affects all rows?

**Comment by Junelson1** [comment]
there is "body-row-class-name" & "body-item-class-name" props to change your row color and even the items inside it, for your reference https://hc200ok.github.io/vue3-easy-data-table-doc/props/common-props.html
